locals.combined_objects.tf

```
@@ -114,6 +114,9 @@ locals {
   combined_objects_postgresql_flexible_servers                    = merge(tomap({ (local.client_config.landingzone_key) = module.postgresql_flexible_servers }), try(var.remote_objects.postgresql_flexible_servers, {}))
   combined_objects_postgresql_servers                             = merge(tomap({ (local.client_config.landingzone_key) = module.postgresql_servers }), try(var.remote_objects.postgresql_servers, {}))
   combined_objects_private_dns                                    = merge(tomap({ (local.client_config.landingzone_key) = module.private_dns }), try(var.remote_objects.private_dns, {}))
+  combined_objects_private_dns_resolvers                          = merge(tomap({ (local.client_config.landingzone_key) = module.private_dns_resolvers }), try(var.remote_objects.private_dns_resolvers, {}))
+  combined_objects_private_dns_resolver_outbound_endpoints        = merge(tomap({ (local.client_config.landingzone_key) = module.private_dns_resolver_outbound_endpoints }), try(var.remote_objects.private_dns_resolver_outbound_endpoints, {}))
```

Add combined_objects_private_dns_resolver_inbound_endpoints as it will be required for a future PR for vnet DNS_SERVERS. We have a mecanism to reference the private ip of an azure firewall, vm or load balancer as the private dns servers. private_dns_resolver_inbound_endpoints will be added to that list and required to be added in the combined objects
```suggestion
  combined_objects_private_dns_resolver_inbound_endpoints        = merge(tomap({ (local.client_config.landingzone_key) = module.private_dns_resolver_inbound_endpoints }), try(var.remote_objects.private_dns_resolver_inbound_endpoints, {}))
  combined_objects_private_dns_resolver_outbound_endpoints        = merge(tomap({ (local.client_config.landingzone_key) = module.private_dns_resolver_outbound_endpoints }), try(var.remote_objects.private_dns_resolver_outbound_endpoints, {}))
  
```